### PR TITLE
Feature/end attempt interaction

### PR DESCRIPTION
**Pull request by ssipasseuth**
Add the standard end attempt interaction in TAO

### CODE REVIEW

File: views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+     * @param {object} interaction
+     */
+    var render = function(interaction, options){
+
+        var $container = Helper.getContainer(interaction);
+
+        //on click, 
+        $container.on('click.commonRenderer', function(){
+            $container.val(true);
+            console.log('trigger submitting', getResponse(interaction));
```
Comment (krampstudio) [inline views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js]: Please remove `console.log`

File: views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+define([
+    'lodash',
+    'tpl!taoQtiItem/qtiCommonRenderer/tpl/interactions/endAttemptInteraction',
+    'taoQtiItem/qtiCommonRenderer/helpers/Helper',
+    'taoQtiItem/qtiCommonRenderer/helpers/PciResponse',
+    'i18n'
+], function(_, tpl, Helper, pciResponse, __){
```
Comment (krampstudio) [inline views/js/qtiCommonRenderer/renderers/interactions/EndAttemptInteraction.js]: Add the `"use strict";` declaration after the main function (same for the other files)

File: views/js/qtiCreator/helper/creatorRenderer.js [hunk 3]
```diff
@@ -9,6 +9,19 @@ define([
     //configure and instanciate once only:
     var _creatorRenderer = null;
     
+    var _configurableInteractions = ['endAttempt'];
```
Comment (krampstudio) [inline views/js/qtiCreator/helper/creatorRenderer.js]: Please comment this section/function so others know what it is used for

**Comment by krampstudio** [comment]
Some files also miss comments/jsdoc

**Comment by krampstudio** [comment]
Let me know when it's ready to merge

**Comment by ssipasseuth** [comment]
Oh sorry, forgot to tell you. 
Yes, it is ready now : I added some jsdoc and added "use strict" on a bunch of files for the item creator source (not every one though). I will do it progressively as I see them in the future.

**Pull request merged**